api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderConstants.java

```diff
@@ -22,7 +22,8 @@ public static final class Apps {
         public static final String ARCHIVES_ROOM = "paperrecord.app.archivesRoom";
         public static final String SYSTEM_ADMINISTRATION = "coreapps.app.systemAdministration";
         public static final String APPOINTMENT_SCHEDULING_HOME = "appointmentschedulingui.app";
-        public static final String DISPENSING = "dispensing.app";
+        public static final String DISPENSING = "dispensing.app";  // This is the legacy, form-based dispensing app
+        public static final String MEDICATION_DISPENSING = "medicationDispensing.app";  // This is the new, MFE-based dispensing app
```

Comment:
Could you also update the [pihemr README](https://github.com/PIH/openmrs-config-pihemr#existing-components)?

Reply:
Thanks - didn't remember that was there @brandones  - done.